ledger-tool/src/bigtable.rs

```diff
@@ -584,6 +830,124 @@ impl BigTableSubCommand for App<'_, '_> {
+                                    "Destination bigtable creds path. (should be writable)",
+                                ),
+                        )
+                        .arg(
+                            Arg::with_name("destination_endpoint")
+                                .long("destination-endpoint")
+                                .value_name("DESTINATION_ENDPOINT")
+                                .takes_value(true)
+                                .help(
+                                    "Destination bigtable endpoint",
+                                ),
+                        )
+                        .arg(
+                            Arg::with_name("destination_is_emulator")
+                            .long("destination-is-emulator")
+                            .value_name("DESTINATION_IS_EMULATOR")
+                            .takes_value(false)
+                            .help(
+                                "Destination is an emulator",
+                            ),
+                        )
+                        .arg(
+                            Arg::with_name("destination_instance_name")
+                                .long("destination-instance-name")
+                                .takes_value(true)
+                                .value_name("DESTINATION_INSTANCE_NAME")
+                                .default_value(solana_storage_bigtable::DEFAULT_INSTANCE_NAME)
+                                .help("Destination bigtable instance name")
+                        )
+                        .arg(
+                            Arg::with_name("destination_app_profile_id")
+                                .long("destination-app-profile-id")
+                                .takes_value(true)
+                                .value_name("DESTINATION_APP_PROFILE_ID")
+                                .default_value(solana_storage_bigtable::DEFAULT_APP_PROFILE_ID)
+                                .help("Destination bigtable app profile id")
+                        )
+                        .arg(
+                            Arg::with_name("starting_slot")
+                                .long("starting-slot")
+                                .validator(is_slot)
+                                .value_name("START_SLOT")
+                                .takes_value(true)
+                                .required(true)
+                                .help(
+                                    "Start copying at this slot",
+                                ),
+                        )
+                        .arg(
+                            Arg::with_name("ending_slot")
+                                .long("ending-slot")
+                                .validator(is_slot)
+                                .value_name("END_SLOT")
+                                .takes_value(true)
+                                .help("Stop copying at this slot [default: starting-slot + 1]"),
```

Comment:
Is this inclusive?

Reply:
I made it inclusive. https://github.com/solana-labs/solana/pull/28122/commits/4b3c3597f1e74c71567ff10964430a2cc614259a it seems more user friendly